bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polymerfire",
-  "version": "0.9.4",
+  "name": "polymerfire-extended",
+  "version": "0.9.5",
   "authors": [
```

Comment:
Ok, sorry. I have removed my changes from bower.json. I think is clean now, ready to be merged.